librz/include/rz_type.h

```diff
@@ -175,6 +175,36 @@ typedef struct rz_type_path_t {
 	char *path;
 } RzTypePath;
 
+/* TODO: what to do with signed/unsigned conditionals? */
```

Comment:
please add doxygen docs to all these "new" structures here. For example, what is this RzTypeCond exactly? Why it fits in RzType?